ideas/92-disc-v5-research.md

```diff
@@ -80,7 +80,25 @@ Description:
 1. It's possible to start `statusd` and Status App with DiscV5 enabled instead of static nodes (switch between them with a flag would be great),
 1. There is an automated way to collect CPU and network usage metrics in an isolated environment.
 
-#### Exit criteria
+### Iteration 1
+
+Goal Date: 2018-05-18
+
+Description:
+- [ ] increase awarness by changing the UI copy that Rinkeby network uses Discovery protocol,
+- [ ] make LES working in Mainnet and Ropsten,
+- [ ] design how mailservers can work in dynamic environment,
+- [ ] make it configurable from the UI/command line to use discovery protocol,
```

Comment:
just to recap. i still think that this is useless. because we either use discovery and stop maintaining list of static peers or we don't and we keep maintaining list of static peers. 

doing both doesn't make sense, as soon as list of static peers will become invalid such fallback option won't work but if we keep it valid all the time - then discovery is not needed.

Reply:
> Our last session proved that it works but at the same time my battery was running down like crazy and there are tons of other edge cases that we need to test and feel comfortable with before we switch it in nightly and alpha releases.

pow was 0.05. i mentioned it during session